cmd/buildah/manifest.go

```diff
@@ -199,6 +202,9 @@ func init() {
 		Example: `buildah manifest inspect mylist:v1.11`,
 		Args:    cobra.MinimumNArgs(1),
 	}
+	flags = manifestInspectCommand.Flags()
+	flags.StringVar(&manifestInspectOpts.authfile, "authfile", auth.GetDefaultAuthFile(), "path of the authentication file. Use REGISTRY_AUTH_FILE environment variable to override")
+	flags.BoolVar(&manifestInspectOpts.tlsVerify, "tls-verify", true, "require HTTPS and verify certificates when accessing the registry. TLS verification cannot be used when talking to an insecure registry.")
```

Comment:
This is actually implemented via `SystemContextFromOptions`?

Fine, it’s consistent with the other subcommands — but it’s fairly hard to find and follow. In some eventual future, consider some design where the flag definition and implementation are collocated.

Reply:
Yeah its done, similarly for all manifests command. I had to implement these so tests could use it.